src/Storage/AdapterPluginManager.php

```diff
@@ -20,86 +19,6 @@
  */
 class AdapterPluginManager extends AbstractPluginManager
 {
-    /** @var array<string,string> */
-    protected $aliases = [
```

Comment:
Same as https://github.com/laminas/laminas-cache/pull/55 could not cherry-pick it due to conflicts